extensions/interactions/FractionInput/directives/oppia-interactive-fraction-input.component.ts

```diff
@@ -132,6 +134,12 @@ export class InteractiveFractionInputComponent implements OnInit, OnDestroy {
     const isAnswerValid = () => this.isAnswerValid();
     this.currentInteractionService.registerCurrentInteraction(
       submitAnswerFn, isAnswerValid);
+
+    setTimeout(
+      () => {
+        let focusLabel: string = this.labelForFocusTarget;
```

Comment:
Since `this.labelForFocusTarget` is a function, you will need to call it in order to get the value right?

```suggestion
        let focusLabel: string = this.labelForFocusTarget();
```

Reply:
No, we don't have to call it as a function as the value is computed and stored  inside the labelForFocusTarget when it is initialised in "@input"  as string.

Reply:
But then the type of `labelForFocusTarget` should be `string`.

Reply:
done. Thanks a lot.